packages/legend-query-builder/src/components/shared/BasicValueSpecificationEditor.tsx

```diff
@@ -272,6 +272,7 @@ const StringPrimitiveInstanceValueEditor = observer(
             options={queryOptions}
             onChange={changeValue}
             value={selectedValue}
+            inputValue={selectedValue.value}
```

Comment:
I think we can just put `value` here instead.
Also, please help me fix the CSS problem with the line height of this input box 🙏